syntax = "proto3";

package pandora;

option java_outer_classname = "OobProto";


service OOB {
  // Share OOB data
  rpc ShareOobData(OobDataRequest) returns (OobDataResponse);
}

// Local Device OOB data.
message OobDataRequest {
  // OOB data Pairing Hash and Randomizer - 32 bytes.
  bytes oob = 1;
}

// Remote Device OOB data.
message OobDataResponse {
  // OOB data Pairing Hash and Randomizer - 32 bytes.
  bytes oob = 1;
}

